tests/foreman/cli/test_repository.py

```diff
@@ -404,10 +403,10 @@ def test_positive_synchronize_yum_repo(self):
                 Repository.synchronize({'id': new_repo['id']})
                 # Verify it has finished
                 new_repo = Repository.info({'id': new_repo['id']})
-                self.assertEqual(new_repo['sync']['status'], 'Finished')
+                self.assertEqual(new_repo['sync']['status'], 'Success')
```

Comment:
That's an example of guessing :) No status was returned before BZ1152237 fix, we designed the tests for 'Finished' status... And it was implemented as 'Success' instead of 'Finished'